### ISSUE TITLE
Ftp.raw question

### ISSUE THREAD

**Issue by RukiaXi** [sergi/jsftp#211]
Hello everyone,

I'm getting a lot of messages in my log following this:
```
Ftp.raw[pass](args): Use Ftp.raw('pass args') instead.
Ftp.raw[feat](args): Use Ftp.raw('feat args') instead.
Ftp.raw[syst](args): Use Ftp.raw('syst args') instead.
Ftp.raw[type](args): Use Ftp.raw('type args') instead.
Ftp.raw[user](args): Use Ftp.raw('user args') instead.
Ftp.raw[quit](args): Use Ftp.raw('quit args') instead.
```

I have no clue why they're showing up, but it's quite annoying. I'm not using debugMode so I'm not sure why this is happening. am I doing something wrong? I'm not using any raw functions except for quit, by using ftp.raw.quit();

Edit: This issue seems to only happen since the new 1.5.5 version. I went back to the version I started developing on, 1.5.3 and that one seems to work fine.

**Comment by davidjkrause** [comment]
I'm seeing this, too, when I got the 1.5.5 version from a straight "npm install".  I backed up to install@1.5.4, and no longer see these warnings on every action.

**Comment by jacobdr** [comment]
See [this commit](https://github.com/sergi/jsftp/commit/84deb7abbda5753ae02a27c8508c291f61ba0684), and then the PR to fix #209

**Comment by bingzheyuan** [comment]
This problem still exists since npm version of jsftp@1.5.5 is different from the one in this repository.
